src/Interfaces/Shared/hco_interface_common.F90

```diff
@@ -330,5 +333,511 @@ SUBROUTINE GetHcoDiagn ( HcoState,       ExtState,  DiagnName, &
     RC = HCO_SUCCESS
 
   END SUBROUTINE GetHcoDiagn
+
+#ifdef FALSE ! the adjoint-relevant code in here has been moved to 
```

Comment:
Did you intend to leave this as FALSE?

Reply:
Sorry, I missed that when I cleaned up the other ifdef FALSEs. This whole subroutine was moved to hco_calc_mod but I left it there as insurance in case that didn't work and then forgot to delete it after I verified the new code. I have commited a change removing both these blocks.